src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketMirroredRepository.java

```diff
@@ -0,0 +1,54 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketMirroredRepository {
```

Comment:
I'm not thrilled with having two `Repository` classes. I wonder if we can either name it differently or roll it up to something more meaningful.

Reply:
It's `MirroredRepository` in Bitbucket and I would like to keep it consistent with Bitbucket naming as we have done for other objects